src/webhook/validation/deprecation.go

```diff
@@ -47,6 +47,54 @@ func deprecatedFeatureFlagDisableMetadataEnrichment(_ *dynakubeValidator, dynaku
 	return warnIfDeprecatedIsUsed(dynakube, dynatracev1beta1.AnnotationFeatureMetadataEnrichment, dynatracev1beta1.AnnotationFeatureDisableMetadataEnrichment)
 }
 
+func deprecatedFeatureFlagActiveGateReadOnlyFilesystem(_ *dynakubeValidator, dynakube *dynatracev1beta1.DynaKube) string {
+	return warnIfDeprecatedFFisUsed(dynakube, dynatracev1beta1.AnnotationFeatureActiveGateReadOnlyFilesystem)
```

Comment:
could we move this into a single validator that just checks if any of the deprecated feature flags is used and returns a warning with all of them? assuming message max length is allows that

Reply:
☝️ agree with @chrismuellner I would prefer to refactor it using a single validator instead of lots of similar functions.